CHANGELOG.md

```diff
@@ -13,6 +11,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 	- Please note that these configuration options are for offline plots ONLY. For configuration options when embedding online plots please see our [embed tutorial](http://help.plot.ly/embed-graphs-in-websites/#step-8-customize-the-iframe).
 - `colors.py` file which contains functions for manipulating and validating colors and arrays of colors
 - 'scale' param in `FF.create_trisurf` which now can set the interpolation on the colorscales
+- animations now work in offline mode. By running `plotly.offline.plot()` and `plotly.offline.iplot()` with a a `fig` with `frames`, the resulting plt will cycle through the figures defined in `frames` either in the browser or in an ipython notebook respectively.
```

Comment:
typo: `plt` to `plot`. 

Maybe we can include a simple example and a link to `plot.ly/python/animations`?